src/dc_chat.rs

```diff
@@ -725,7 +725,7 @@ unsafe fn prepare_msg_raw(
                             timestamp,
                             (*msg).type_0,
                             (*msg).state,
-                            if !(*msg).text.is_null() { Some(as_str((*msg).text)) } else { None },
+                            (*msg).text.deref().unwrap_or(""),
```

Comment:
@dignifiedquire previously said that using unstable features is discouraged because the goal is to move to stable Rust. This seems to be the line that requires `inner_deref`.

Reply:
Okay.